src/style/_variables.scss

```diff
@@ -232,3 +241,29 @@ $topbar-download-url-button-font-color: $white !default;
 $text-body-default-font-color: $bright-gray !default;
 $text-code-default-font-color: $bright-gray !default;
 $text-headline-default-font-color: $bright-gray !default;
+
+// Fonts
+$special-text-font-family:'Roboto-Medium';
```

Comment:
I don't feel great about using `special` as a variable name unless it really nails what's going on.

**Question**: what should "special" mean to me in this context? Any ideas on alternative names?

Reply:
renamed to `alt`